converter/solid_figures.py

```diff
@@ -35,8 +42,17 @@ class CylinderFigure(SolidFigure):
     radius_bottom: float = 1.
     height: float = 1.
 
+    def expand(self, expansion: float) -> None:
```

Comment:
Main purpose of the Monte-Carlo simulation is particle transport, think of it as of ray tracing.
Think you are a ray. If we'd switch to `margin` concept, then having a margin of say 1cm, means that if you escape the body along a vector normal to its surface (i.e. in locally perpendicular direction), then you have additional 1 cm to travel.

Therefore I suggest renaming `expansion` (which resembles multiplicative factor) to `margin` (which sounds more additive) and changing the code. For radius we do `+ 1 * margin`, for `height` we do ` + 2 * margin`.
Then from the perspective of the ray this would be an additional `margin` to travel.